dist/PathTools/Cwd.pm

```diff
@@ -155,6 +155,8 @@ if($^O ne 'MSWin32') {
     foreach my $try ('/bin/pwd',
 		     '/usr/bin/pwd',
 		     '/QOpenSys/bin/pwd', # OS/400 PASE.
+		     '/run/current-system/profile/bin/pwd', # GUIX
+		     '/run/current-system/sw/bin/pwd' # NixOS
```

Comment:
Importantly, Nix's concept of closures is very important and highly valued. It is very common to export a closure and run it elsewhere, where `/run/` doesn't exist in any form. This works "sometimes" but intentionally sidesteps the design philosophy of Nix, creating software artifacts that appear to work in development and then fails on deployment due to cheating like this.